pkg/expect/expect.go

```diff
@@ -193,7 +193,7 @@ func (ep *ExpectProcess) ExpectFunc(ctx context.Context, f func(string) bool) (s
 
 		select {
 		case <-ctx.Done():
-			return "", fmt.Errorf("failed to find match string: %w", ctx.Err())
+			return "", fmt.Errorf("context done before to found matching log")
```

Comment:
Sentence is a bit hard to understand, perhaps:
```suggestion
			return "", fmt.Errorf("context done before matching log found")
```